libs/product/driver/magento/src/queries/fragments/product.ts

```diff
@@ -28,18 +28,6 @@ export const magentoProductFragment = gql`
 		description {
 			html
 		}
-    upsell_products {
-      ...magentoProductPreview
-      ...magentoBundledProduct
-      ...magentoSimpleProduct
-      ...magentoConfigurableProduct
-    }
-    related_products {
-      ...magentoProductPreview
-      ...magentoBundledProduct
-      ...magentoSimpleProduct
-      ...magentoConfigurableProduct
-    }
```

Comment:
This doesn't seem right. The `magentoConfigurableProduct` fragment should be removed everywhere but nothing else should change.

Reply:
yea I committed this accidentally. Reverted